tests/bom/bomqty.yml

```diff
@@ -0,0 +1,65 @@
+connectors:
+  X1:
+    type: No additional components
+    pincount: 6
+
+  X2:
+    type: Contains additional components
+    pincount: 6
+    additional_components:
+      -
+        type: One, no unit
+      -
+        type: Two kilometers
+        qty: 2 km
+      -
+        type: Takes pincount times seven
+        qty: 7
+        qty_multiplier: pincount
+      -
+        type: Takes 10 mm per populated pin
+        qty: 10 mm
+        qty_multiplier: populated
+      -
+        type: Takes number of connections
+        qty_multiplier: connections
+
+cables:
+  C1:
+    type: Containts additional components
+    wirecount: 4
+    length: 1.5
+    color_code: DIN
+    additional_components:
+      -
+        type: One
+      -
+        type: Three centimeters
+        qty: 3 cm
+      -
+        type: Takes wirecount times two
+        qty: 2
+        qty_multiplier: wirecount
+      -
+        type: Takes length times three
+        qty: 3 # adding unit here should cause error because the length already has a unit
+        qty_multiplier: length
```

Comment:
The comment "adding unit here should cause error because the length already has a unit" made me wonder how to cope with a use case where I would like to add 5 cable clips per meter as `additional_components` to the cable. Is that possible?

Reply:
I had not considered the use case of "items per unit of cable length" as opposed to "length as multiple of cable length". I suggest opening a separate issue with this feature request. Ideally, solid unit handling should be implemented first. Then one could even specify cable length in one unit (e.g. km), and the multiplier in another (e.g. 10 per m), with the unit module handling conversion and cancellation of units...

Reply:
Such a use case was already described in https://github.com/wireviz/WireViz/pull/115#discussion_r495474219 - and was intentionally enabled then. It is still currently supported in the latest release v0.3.2:
```yaml
connectors:
  X1: &x1
    pincount: 3
  X2: *x1
cables:
  W1:
    wirecount: 3
    colors: [BU,BN,YEGN]
    length: 3 m
    additional_components:
      - type: Clips every 20 cm
        qty: 5
        qty_multiplier: length
connections:
  -
    - X1: [1-3]
    - W1: [1-3]
    - X2: [1-3]
```
![cable_clips_html](https://github.com/wireviz/WireViz/assets/12657033/91e515cc-dc4b-4630-99e4-40c97ca8f08e)